src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -652,10 +706,22 @@ public override void Write(string value)
                 Dbg.Assert(writer == _parent.OutputSerializer.textWriter, "writers should be the same");
 
                 _parent.OutputSerializer.Serialize(value);
+
+                if (newLine)
+                {
+                    _parent.OutputSerializer.Serialize(Crlf);
```

Comment:
Add the `Crlf` to value may be better than calling `Serialize` twice.

Reply:
Done.